creator/users/schema.py

```diff
@@ -93,6 +95,12 @@ def _get_study_id(self, info):
 class UserNode(DjangoObjectType):
     display_name = graphene.String(source="display_name")
 
+    organizations = DjangoFilterConnectionField(
```

Comment:
this is good - can we also add `users` sub query to an `organization`?

Reply:
sure, i'll make an issue and work on it and try to get it out today